Math/NumberTheory/ArithmeticFunctions/Standard.hs

```diff
@@ -97,6 +99,22 @@ divisorsHelperSmall p 1 = IS.singleton p
 divisorsHelperSmall p a = IS.fromDistinctAscList $ p : p * p : map (p ^) [3 .. wordToInt a]
 {-# INLINE divisorsHelperSmall #-}
 
+-- | See `divisorsInRangeA`
+divisorsTo :: (UniqueFactorisation n, Ord n) => n -> n -> Set n
+divisorsTo to = runFunction (divisorsToA to)
+
+-- | The set of all (positive) divisors strictly below `to` 
+divisorsToA :: (UniqueFactorisation n, Ord n) => n -> ArithmeticFunction n (Set n)
```

Comment:
It is worth to add a `Integral n` constraint, because this function does not make sense for `GaussianInteger` or `EisensteinInteger`.